src/main/scala/uk/gov/hmrc/hmrcfrontend/config/AccountMenuConfig.scala

```diff
@@ -0,0 +1,48 @@
+/*
+ * Copyright 2021 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package uk.gov.hmrc.hmrcfrontend.config
+
+import play.api.Configuration
+import javax.inject.Inject
+
+class AccountMenuConfig @Inject() (configuration: Configuration) {
+
+  private val perTaxFrontendHost: Option[String] =
+    configuration.getOptional[String]("pta-account-menu.pertax-frontend.host")
+
+  private def baseUrl: String = perTaxFrontendHost.getOrElse("")
+
+  val accountHomeHref: String = {
+    val accountMenuHost = configuration.getOptional[String]("pta-account-menu.accountHome.host").getOrElse(baseUrl)
+    s"$accountMenuHost${configuration.get[String]("pta-account-menu.accountHome.path")}"
+  }
```

Comment:
```suggestion
  val accountHomeHref: String = 
    s"$configuration.get[String]("pta-account-menu.account-home.host")${configuration.get[String]("pta-account-menu.account-home.path")}"
```
example of how it could simplify this, so remove the default values in code and just use them in the config